HelpSource/Classes/File.schelp

```diff
@@ -111,6 +111,27 @@ method::mtime
 Get last modification time in seconds since the Epoch.
 returns:: an link::Classes/Integer::
 
+subsection:: Error handling in filesystem utilities
+
+If one of the above file system primitives fails, in most cases, a PrimitiveFailedError object will be thrown:
+
+code::
+File.mkdir("/usr/oh-no-you-cant");
+
+ERROR: Primitive '_FileMkDir' failed.
+caught exception 'boost::filesystem::create_directory: Permission denied: "/usr/oh-no-you-cant"' in primitive in method Meta_File:mkdir
```

Comment:
this gets sc syntax highlighting when it shouldn't; you can use a `teletype::` block instead to avoid that